src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -28,6 +29,11 @@ class CrudField {
             if(this.input.length === 0) {
                 this.input = this.wrapper.find('input, textarea, select').first();
             }
+
+            // Validate that the field has been found
+            if(this.wrapper.length === 0 || this.input.length === 0) {
+                console.error(`CrudField "${this.name}" was not found.`);
+            }
```

Comment:
This is a super handy validation 👌
It will **warn** developers when they are trying to use a non existing field.
Until now it just doesn't work, without any warnings.